repair-service/service/src/main/java/com/epam/repair/rest_app/AddressServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package com.epam.repair.rest_app;
+
+import com.epam.repair.dao.AddressDao;
+import com.epam.repair.model.Address;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+/**
+ * Address Service Interface implementation.
+ */
+@Service
+@Transactional
+public class AddressServiceImpl implements AddressService{
```

Comment:
checkstyle should fails here.
guess why?

Reply:
Comment accepted. Package name changed.